nav2_map_server/src/map_io.cpp

```diff
@@ -202,8 +207,9 @@ void loadMapFromFile(
   Magick::InitializeMagick(nullptr);
   nav_msgs::msg::OccupancyGrid msg;
 
-  std::cout << "[INFO] [map_io]: Loading image_file: " <<
-    load_parameters.image_file_name << std::endl;
+  RCLCPP_INFO_STREAM(
+    rclcpp::get_logger("map_io"), "Loading image_file: " <<
```

Comment:
You can reduce the boilerplate by creating a Logger object at the start of the methods that have logging and pass those in instead of recreating each time